WordPress/WordPressTest/PostTests.swift

```diff
@@ -376,6 +376,13 @@ class PostTests: XCTestCase {
         XCTAssertFalse(revision.hasLocalChanges())
     }
 
+    func testThatVersionConflictWorks() {
+        #warning("create tests")
```

Comment:
Here we want two tests: one that given the conditions `hasVersionConflict` is `true` and another that is false.

What we want to do is to have a post that has an `original` and manipulate their dates, that way we'll ensure the logics of `hasVersionConflict`. Using .revision() [like here](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/WordPressTest/ViewRelated/Post/Views/PostCardStatusViewModelTests.swift#L96) might work. :)